src/views/states/list/components/InterfacesTypeSection.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+
+import { ExpandableSectionToggle, List, ListItem, Tooltip } from '@patternfly/react-core';
+import { Td, Tr } from '@patternfly/react-table';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+import './interfaces-table.scss';
+
+interface InterfacesTypeSectionProps {
+  interfaces: NodeNetworkConfigurationInterface[];
+  interfaceType: string;
+}
+
+const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = ({ interfaceType, interfaces }) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+
+  return (
+    <>
+      <Tr className="interfaces-table__interface-type-row">
+        <Td colSpan={6}>
+          <ExpandableSectionToggle isExpanded={isExpanded} onToggle={setIsExpanded}>
+            <strong className="pf-u-color-100">{interfaceType}</strong>
```

Comment:
I'd recommend to style the text and color not here, but in a css file, and using `font-weight` instead of <strong>.

Reply:
I would create a `scss` file just for that...

Reply:
I'd go for it as it would make this code cleaner. With time this code can contain more stuff like that and so it can become less readable very easily, and then it will be hard to change that...